pytest_tests/tests/services/rest_gate/test_rest_gate.py

```diff
@@ -7,19 +7,19 @@
 import pytest
 from helpers.container import create_container
 from helpers.file_helper import generate_file, generate_file_with_content
+from helpers.neofs_verbs import put_object_to_random_node
 from helpers.rest_gate import (
     attr_into_header,
     get_object_by_attr_and_verify_hashes,
     get_via_rest_gate,
+    quote,
     try_to_get_object_and_expect_error,
     upload_via_rest_gate,
-    quote,
 )
-from helpers.neofs_verbs import put_object_to_random_node
 from helpers.utility import wait_for_gc_pass_on_storage_nodes
 from helpers.wellknown_acl import PUBLIC_ACL
-from rest_gw.rest_utils import get_object_and_verify_hashes
 from neofs_env.neofs_env_test_base import NeofsEnvTestBase
+from rest_gw.rest_utils import get_object_and_verify_hashes
```

Comment:
Formatting change, not related to marks.

Reply:
removed